### PR TITLE
add:  Snackbar Component

### PR DESCRIPTION
**Pull request by kavyakunder**
Snackbar Component

https://user-images.githubusercontent.com/81958043/153041783-ae80b863-5516-46c6-890f-7bb5cd02db32.mp4

### CODE REVIEW

File: Components/Snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.snackbar {
+  text-align: center;
+  padding: 20px 20px;
+  background-color: var(--secondary-color);
+  color: var(--white);
+  margin: 5px;
+  border-radius: 5px;
+}
```
Comment (shuklahimanshu81) [inline Components/Snackbar/snackbar.css]: Suggestion:  Create a variable for border-radius because you have used same unit at multiple elements.

**Review by shuklahimanshu81 — COMMENTED** [review]
Everything else looks good to me. Can be merged.

File: Components/Snackbar/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  color: var(--primary-color);
+  cursor: pointer;
+  border-radius: 5px;
+}
+
+.snackbar-icon {
+  align-items: center;
+  text-align: center;
+  border-radius: 100%;
+  padding: 10px;
```
Comment (AnuragSahu11) [inline Components/Snackbar/snackbar.css:50]: You can create variables for sizes like 10px etc which you have used many times in padding, margins

**Review by AnuragSahu11 — COMMENTED** [review]
Looks good

File: Components/Snackbar/snackbar.css (hunk 1 above)
Comment (kavyakunder) [reply Components/Snackbar/snackbar.css]: done!

File: Components/Snackbar/snackbar.css (hunk 2 above)
Comment (kavyakunder) [reply Components/Snackbar/snackbar.css:50]: I have done for 5px which is used more frequently.

**Pull request merged**